principles.qmd

```diff
@@ -119,6 +119,15 @@ Consistent, high quality, and accessible documentation is key to adopting softwa
 - Aim to provide translations for the most common languages
 - We aim to provide [automated blog posts for major and minor releases](https://epiverse-trace.github.io/blog.html#category=new-release). Code owners may occasionally decide to release patch blogs if the patch is critical.
 
+### Website modes
+
+Epiverse-TRACE packages use [`pkgdown`](https://pkgdown.r-lib.org/) to generate websites. `pkgdown` provides [options](https://pkgdown.r-lib.org/reference/build_site.html#development-mode) (in the `_pkgdown.yml` file) to generate different sites for the development and released versions of your package.
+
+When the package is not yet on CRAN, we use the `unreleased` mode. After the package is on CRAN, we switch to `auto`, which provides two versions of the website: `release` and `devel`. This is because we expect most users to install from CRAN and so we direct them to the frozen release site, with an option to manually switch to the `devel` website if they installed from GitHub.
```

Comment:
Please provide an example package that is using `unreleased` and hyperlink it and the same for a package using `auto`. I always find it helps to explicitly see these setting being used in actual packages.

Reply:
Thanks, Josh. I've incorporated your changes. Since the contents have moved, I'm unable to link the commits so, it'll be better to re-review.